src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacSymbolicChemistryValidator.java

```diff
@@ -191,6 +191,8 @@ public QuestionValidationResponse validateQuestionResponse(final Question questi
                     req.put("target", formulaChoice.getMhchemExpression());
                     req.put("test", submittedFormula.getMhchemExpression());
                     req.put("description", chemistryQuestion.getId());
+                    req.put("allowPermutations", String.valueOf(chemistryQuestion.allowPermutations()));

```

Comment:
What happens if this is null? Will this produce a string `"null"` rather than `null`?

Reply:
(I think we decided that [the checker won't notice either way](https://github.com/isaacphysics/chemistry-checker-js/pull/1/files#diff-0313b3dba60dab2976642eac726c4a1ce1c8d222ad198a13097fbc0c9aa038f5R29), since it expects the string `"true"`).